Tests/DangerTests/GitLabMilestoneTests.swift

```diff
@@ -0,0 +1,37 @@
+@testable import Danger
+import DangerFixtures
+import XCTest
+
+class GitLabMilestoneTests: XCTestCase {
+    func testParseGroupMilestone() {
+        let jsonString = DSLGitLabGroupMilestoneJSON
+        let data = Data(jsonString.utf8)
+
+        do {
+            let milestone = try jsonDecoder.decode(GitLab.MergeRequest.Milestone.self, from: data)
+            XCTAssertEqual(milestone.groupId, 9_449_452)
+            XCTAssertNil(milestone.projectId)
+        } catch {
+            XCTFail(error.localizedDescription)
+        }
```

Comment:
If you make the test `throw` it will do this handling automatically

 ```suggestion
        let milestone = try jsonDecoder.decode(GitLab.MergeRequest.Milestone.self, from: data)
        XCTAssertEqual(milestone.groupId, 9_449_452)
        XCTAssertNil(milestone.projectId)
```

Reply:
ow, make it throwable is better, thanks for the suggestion!